text/0772-deprecate-bower-support.md

```diff
@@ -1,10 +1,9 @@
 ---
-Stage: Accepted
+Stage: Ready for Release
 Start Date: 2021-10-21
 Release Date: Unreleased
 Release Versions:
-  ember-source: vX.Y.Z
-  ember-data: vX.Y.Z
+  ember-cli: v4.3.0
```

Comment:
ember-cli v4.3 has not been released yet.

Reply:
@locks it was in beta already, so it's in sort of "almost there" state. removed for now